security/pfSense-pkg-acme/files/usr/local/www/acme/acme_certificates.php

```diff
@@ -0,0 +1,378 @@
+				<td>
+					<input type="checkbox" id="frc<?=$certificatename;?>" onClick="fr_toggle('<?=$certificatename;?>')" name="rule[]" value="<?=$certificatename;?>"/>
+					<a class="fa fa-anchor" id="Xmove_<?=$certificatename?>" title="<?=gettext("Move checked entries to here")?>"></a>
+				</td>
+			  <td>
+				<?php
+					if ($certificate['status']=='disabled'){
+						$iconfn = "disabled";
+					} else {
+						$iconfn = "enabled";
+					}?>
+				<a id="btn_<?=$certificatename;?>" href='javascript:togglerow("<?=$certificatename;?>");'>
+					<?=acmeicon($iconfn, gettext("click to toggle enable/disable this certificate renewal"))?>
+				</a>
+			  </td>
+			  <td>
+				<?=$certificate['name'];?>
+			  </td>
+			  <td>
+				<?=$certificate['desc'];?>
+			  </td>
+			  <td>
+				<?=$certificate['acmeaccount'];?>
+			  </td>
+			  <td>
+				<?=date('d-m-Y H:i:s', $certificate['lastrenewal']);?>
+			  </td>
+			  <td>
+				  <?php
+					$method = "";
+					if (is_array($certificate['a_domainlist']['item'])) {
+						foreach($certificate['a_domainlist']['item'] as $domain) {
+							if ($domain['status'] == 'disable') {
+								continue;
+							}
+							$method = $domain['method'];
+						}
+					}
+			
+				  if ($method == "dns_manual"): ?>
+				  <a href='javascript:renewcertificate("<?=$certificatename;?>");' class="btn btn-sm btn-primary">
+					  <i id="btnrenewicon_<?=$certificatename;?>" class="fa fa-check"></i> Renew
+				  </a>
+				  <a href='javascript:issuecertificate("<?=$certificatename;?>");' class="btn btn-sm btn-primary">
+					  <i id="btnissueicon_<?=$certificatename;?>" class="fa fa-check"></i> Issue
+				  </a>
+				  <?php else: ?>
+				  <a href='javascript:issuecertificate("<?=$certificatename;?>");' class="btn btn-sm btn-primary">
+					  <i id="btnissueicon_<?=$certificatename;?>" class="fa fa-check"></i> Issue/Renew
+				  </a>
+				  <?php endif; ?>
+			  </td>
+			  <td class="action-icons">
+				<button style="display: none;" class="btn btn-default btn-xs" type="submit" id="move_<?=$certificatename?>" name="move_<?=$certificatename?>" value="move_<?=$certificatename?>"></button>
+				<a href="acme_certificates_edit.php?id=<?=$certificatename;?>">
+					<?=acmeicon("edit", gettext("edit"))?>
+				</a>
+				<a href="acme_certificates.php?act=del&amp;id=<?=$certificatename;?>" onclick="return confirm('Do you really want to delete this entry?')">
+					<?=acmeicon("delete", gettext("delete"))?>
+				</a>
+				<a href="acme_certificates_edit.php?dup=<?=$certificatename;?>">
+					<?=acmeicon("clone", gettext("clone"))?>
+				</a>
+			  </td>
+			</tr><?php
+		}
+?>				
+				</tbody>
+			</table>
+		</div>
+	</div>
+	<nav class="action-buttons">
+		<a href="acme_certificates_edit.php" role="button" class="btn btn-sm btn-success" title="<?=gettext('Add backend to the end of the list')?>">
+			<i class="fa fa-level-down icon-embed-btn"></i>
```

Comment:
Single add buttons should use fa-plus, not fa-level-down. Only use fa-level-down when the page supports adding to the top or bottom of a list (For example, when there is also an fa-level-up add button).